### PR TITLE
Feature: Add README

### PR DESCRIPTION
**Pull request by maxhoheiser**
Add Readme and code of conduct

### CODE REVIEW

File: README.md
Comment (joe4dev) [inline README.md]: that's not our official Homebrew package. We have our own tap: https://docs.localstack.cloud/getting-started/installation/#localstack-cli

File: README.md
Comment (joe4dev) [inline README.md]: The verbose installation instructions can be nice for completeness but are macOS-opinionated. We usually have a bullet list with links. Example: https://github.com/localstack-samples/sample-shipment-list-demo-lambda-dynamodb-s3?tab=readme-ov-file#prerequisites

File: README.md
Comment (joe4dev) [inline README.md]: typo: "cosnequitive"

File: README.md
Comment (joe4dev) [inline README.md]: style: I generally recommend starting each sentence on a new line for better git diffs and diff readability. Like here for the first sentence 👍

File: README.md
Comment (joe4dev) [inline README.md:118]: I appreciate a standardized Makefile for automation (despite its terrible syntax) 👍

File: README.md
Comment (joe4dev) [inline README.md:93]: Where? being more concrete here would help

File: README.md
Comment (joe4dev) [inline README.md:87]: How?

File: README.md
Comment (joe4dev) [inline README.md:88]: Can we be more specific what are the minimal changes required? Can I just `cp .env.example .env`?
Let's try to make it as simple as possible for the user

File: README.md
Comment (joe4dev) [inline README.md:135]: All Makefile targets should probably be`.PHONY`

File: README.md
Comment (joe4dev) [inline README.md:142]: Very valuable to have an automated test scenario 👍

File: README.md
Comment (joe4dev) [inline README.md:145]: nit: Running the test yields a warning:

```
  /Users/joe/Projects/LocalStack-Samples/sample-cdk-kinesis-firehose-redshift/utils/helper_methods.py:76: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
    records = pd.read_sql(sql_query, conn)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```

A comment or warning suppression could be helpful if we don't want to address it

File: README.md
Comment (joe4dev) [inline README.md]: Very nice demo using Jupyter Notebook 👏👏👏

Comment (joe4dev) [inline README.md]: It could be worth mentioning how to use it. Open how? The VSCode integration can be quite useful here. It just requires creating a new environment (i.e., adding the existing venv). You could also link to the VSCode docs for further usage hints: https://code.visualstudio.com/docs/datascience/jupyter-notebooks

**Review by joe4dev — APPROVED** [review]
LGTM after minor modifications 👍 

Please clarify which version of LocalStack is required before merging.
If LocalStack Pro is required, this needs to be very prominent and also affects the docs how we start LocalStack. The auth token should be mentioned there.

File: README.md
Comment (maxhoheiser) [reply README.md]: fixed in: https://github.com/localstack-samples/sample-cdk-kinesis-firehose-redshift/pull/4/commits/995e825c76bc5911c61d1c6b0296724e07115eb0

File: README.md
Comment (maxhoheiser) [reply README.md]: added general section and moved this to toggle in: https://github.com/localstack-samples/sample-cdk-kinesis-firehose-redshift/pull/4/commits/995e825c76bc5911c61d1c6b0296724e07115eb0

File: README.md
Comment (maxhoheiser) [reply README.md:87]: added detailes in: https://github.com/localstack-samples/sample-cdk-kinesis-firehose-redshift/pull/4/commits/8f4948368254b66a405a519cd7c7b2a229433efd

File: README.md
Comment (maxhoheiser) [reply README.md:88]: added explanation in: https://github.com/localstack-samples/sample-cdk-kinesis-firehose-redshift/pull/4/commits/2a252b055cc3255710866f3fc744b1f87e6657a7
and updated .env.examples to make it directly usable in: https://github.com/localstack-samples/sample-cdk-kinesis-firehose-redshift/pull/4/commits/cdc5bbca21b2ffff4e36a2ce31bc0308dbb9ed5c

File: README.md
Comment (maxhoheiser) [reply README.md:135]: added in: https://github.com/localstack-samples/sample-cdk-kinesis-firehose-redshift/pull/4/commits/694deeab49cbd3791764a3b7c3bfd076db4ed0d9

File: README.md
Comment (maxhoheiser) [reply README.md]: fixed in: https://github.com/localstack-samples/sample-cdk-kinesis-firehose-redshift/pull/4/commits/b8422b5f9f87449ef1512a7dded61be051c101ed

File: README.md
Comment (maxhoheiser) [reply README.md]: added expalnation in: https://github.com/localstack-samples/sample-cdk-kinesis-firehose-redshift/pull/4/commits/b4a67cddd8f321d027f63fd8993ebd7c92bd3e4b

**Pull request merged**
